cnf/ext/central.mvn

```diff
@@ -1,6 +1,7 @@
 org.osgi:osgi.annotation:7.0.0
 org.osgi:org.osgi.core:4.2.0
 org.osgi:osgi.core:6.0.0
+org.osgi:osgi.cmpn:6.0.0
```

Comment:
Please DO NOT USE osgi.cmpn. I put some effort into ridding the build of osgi.cmpn. Use the individual jars so we have proper control over the versions of individual components used.

Thanks.